cumulusci/tasks/datadictionary.py

```diff
@@ -28,44 +54,139 @@ class GenerateDataDictionary(BaseGithubTask):
         "field_path": {
             "description": "Path to a CSV file to contain an field-level data dictionary."
         },
-        "release_prefix": {
-            "description": "The tag prefix used for releases.",
-            "required": True,
+        "include_dependencies": {
+            "description": "Process all of the GitHub dependencies of this project and include their schema in the data dictionary.",
+            "default": True,
+        },
+        "additional_dependencies": {
+            "description": "Include schema from additional GitHub repositories that "
+            "are not explicit dependencies of this project to build a unified data dictionary. Specify as a list of URLs."
         },
     }
 
+    def _init_task(self):
+        # Walking GitHub releases seems to be especially prone to API timeouts.
+        # Increase the defaults substantially to try to cope.
+        self.project_config.github_session = GitHubSession(
+            default_read_timeout=30, default_connect_timeout=30
+        )
+        super()._init_task()
```

Comment:
Why not always use this customized session? Would simplify things a bit.

Reply:
Sorry, I'm not sure I follow the alternative.

Reply:
@davidmreed I'm just saying, these customized timeouts feel like something that we could apply to all github3 clients constructed by cci, rather than just for this task.

Reply:
Oh, got it. I'm fine with that - I was aiming just to reduce the scope of intervention and testing but I suppose increasing the timeout is pretty unlikely to produce any regression behavior.

Reply:
Revised to use the longer timeout everywhere unless explicitly overridden.